sdk/managedserviceidentity/Azure.ResourceManager.ManagedServiceIdentities/CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
+## 2.0.0 (2023-02-20)
```

Comment:
For track2 mgmt SDK, we do not allow bumping the major version which normally indicates breaking change. Please change to `1.1.0`

Reply:
changed to 1.1.0